devicetypes/smartthings/cree-bulb.src/cree-bulb.groovy

```diff
@@ -15,14 +15,15 @@
  */
 
 metadata {
-    definition (name: "Cree Bulb", namespace: "smartthings", author: "SmartThings") {
+    definition (name: "Cree Bulb", namespace: "smartthings", author: "SmartThings", category: "C6") {
```

Comment:
I'd think this would be `C1` because essentially a lightbulb is like a ZigBee Switch. Perhaps @tpmanley can clarify?